src/components/home/Achievements.tsx

```diff
@@ -5,7 +5,7 @@ import { createAuthors } from '../../models/creator'
 import { GQLData } from '../../models/gql'
 import { SectionBlack } from '../shared'
 
-import styles from './Achievements.module.scss'
+import * as styles from './Achievements.module.scss'
```

Comment:
We should switch to using `import { someClassName } from './Achievements.module.scss'`. However, we'd need to help TypeScript to do so first.